packages/datagrid/src/components/__snapshots__/datagrid.test.js.snap

```diff
@@ -0,0 +1,268 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`#DataGrid should render DefaultDateCellRenderer 1`] = `
+<div
+  className="theme-td-grid theme-undefined td-grid"
+>
+  <AgGridReact
```

Comment:
The snapshots doesn't reflect what you want to test, shallow rendering is not what you need, try mount